styles/main.css

```diff
@@ -35,10 +37,12 @@ a {
 }
 
 header nav {
-	margin-left: -2em;
-	margin-right: -2em;
+	margin-left: 2em;
+	margin-right: 2em;
 	border-bottom: 1px dotted rgba(0,0,0,0.3);
 	margin-top: -4px;
+
+    overflow: auto;

```

Comment:
This is the declaration that makes the nav element take on the height of its floated contents